implementation/src/main/java/io/smallrye/mutiny/groups/UniMemoize.java

```diff
@@ -50,9 +50,27 @@ public Uni<T> until(BooleanSupplier invalidationGuard) {
      *        {@code null}, must be strictly positive
      * @return a new {@link Uni}
      * @apiNote This is an experimental API
+     * @deprecated use {@link forDuration(Duration)} instead
```

Comment:
It's not enough to deprecate in javadocs, one also needs to add a deprecation annotation to a method.